fmn/delivery/backends/mail.py

```diff
@@ -0,0 +1,131 @@
+from .base import BaseBackend
+
+from kitchen.text.converters import to_bytes
+
+import email
+import logging
+
+from twisted.internet import defer
+from twisted.mail import smtp
+
+from fmn.lib import models
+from fmn.util import get_fas_email
+
+_log = logging.getLogger(__name__)
+
+
+CONFIRMATION_TEMPLATE = u"""
+{username} has requested that notifications be sent to this email address
+* To accept, visit this address:
+  {acceptance_url}
+* Or, to reject you can visit this address:
+  {rejection_url}
+Alternatively, you can ignore this.  This is an automated message, please
+email {support_email} if you have any concerns/issues/abuse.
+"""
```

Comment:
This template looks a lot like the one above in another file. Are they different? If not, maybe they could share a variable.